src/main/java/eu/webeid/security/validator/ocsp/OcspClientImpl.java

```diff
@@ -66,31 +62,36 @@ public static OcspClient build(Duration ocspRequestTimeout) {
      */
     @Override
     public OCSPResp request(URI uri, OCSPReq ocspReq) throws IOException {
-        final RequestBody requestBody = RequestBody.create(ocspReq.getEncoded(), OCSP_REQUEST_TYPE);
-        final Request request = new Request.Builder()
-            .url(uri.toURL())
-            .post(requestBody)
+        final HttpRequest request = HttpRequest.newBuilder()
+            .uri(uri)
+            .header("Content-Type", OCSP_REQUEST_TYPE)
```

Comment:
Can you use here also static variable?

Reply:
Good point, thanks! I will add the static variable.

Reply:
Done.